src/etos_test_runner/etr.py

```diff
@@ -138,14 +142,54 @@ def load_plugins(self):
             plugins.append(module.ETRPlugin(self.etos))
         self.etos.config.set("plugins", plugins)
 
+    def get_sub_suite_url(self, environment_id):
+        """Get sub suite from ETOS environment defined event.
+
+        :param environment_id: ID of th environment defined event.
+        :type environment_id: str
+        :return: URL for sub suite.
+        :rtype: str
+        """
+        query = (
+            """
+        {
+          environmentDefined(search: "{'meta.id': '%s'}") {
+            edges {
+              node {
+                uri
+              }
+            }
+          }
+        }
+        """
+            % environment_id
+        )
+        wait_generator = self.etos.utils.wait(self.etos.graphql.execute, query=query)
+        for response in wait_generator:
+            if response:
+                try:
+                    _, environment_defined = next(
+                        self.etos.graphql.search_for_nodes(response, "environmentDefined")
+                    )
+                except StopIteration:
+                    return None
+                _LOGGER.info(environment_defined)
```

Comment:
A tad more context to this log print would be nice.

Reply:
```suggestion
```